types/react-fontawesome/react-fontawesome-tests.tsx

```diff
@@ -14,3 +14,14 @@ class TestComponent extends React.Component {
     );
   }
 }
+
+class CSSTestComponent extends React.Component {
+  render() {
+    return (
+      <FontAwesome
+        name="rocket"
+        style={{backgroundColor: 'red', color: 'blue'}}
+      />
+    );
+  }
+}
```

Comment:
I ran the tests, and it asks for a new line at the end. Try running the tests and see if it passes